include/spblas/vendor/rocsparse/multiply.hpp

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <type_traits>
+
+#include <hip/hip_runtime.h>
+#include <rocsparse/rocsparse.h>
+
+#include <spblas/detail/ranges.hpp>
+#include <spblas/detail/view_inspectors.hpp>
+
+#include "exception.hpp"
+#include "hip_allocator.hpp"
+#include "types.hpp"
+
+namespace spblas {
+
+class spmv_state_t {
+public:
```

Comment:
using "this->" will significantly improve readability.

Reply:
do you mean also for the data?
I am used to use `this->` only for function.

Reply:
Coding style. I use this-> whenever possible.